app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/Produsent.kt

```diff
@@ -103,6 +104,12 @@ object Produsent {
                 }
                 httpServer.start(wait = true)
             }
+            launch {
+                val roller = altinn.hentRoller()
```

Comment:
tror poden dør dersom vi får feil mot altinn slik det er nå. 
for å unngå det kan man hindre at exception fra hentRoller bobler opp.

Reply:
Man kunne kanskje vurdert å kjøre dette i en loop? Med delay på f.eks. 24. time. Så er man ikke avhengig av en redeploy i tilfelle det kommer ut nye roller som noen ønsker å bruke.